pre-post/check-who.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
For the folders, pre and post are different. Could be a good pattern to have a folder for the pre and a folder for the post decorators?